### ISSUE TITLE
SPMC-17.6a2 Audio Passthrough Issues

### ISSUE THREAD

**Issue by nvlashi** [koying/SPMC#1184]
I can't see a way to pass through AC3, DTS, & E-AC3 with the audio passthrough settings on the ARM32 build of SPMC-17.6a2.  Here's the behaviour I'm seeing:

Audio Passthrough
 - Passthrough output device
 	If I select: Anroid RAW, Passthrough, I see the following
 		Dolby Digital (AC3) capable receiver
 		Dolby Digital Plus (E-AC3) capable receiver

	If I select Android IEC, Passthrough, I see the following
 		Dolby Digital (AC3) capable receiver
		DTS Capable Receiver
Is this expected behavior or a bug because in the 16.7.3 build, you had options for enabling a receiver with all 3 (AC3, E-AC3, & DTS).

**Comment by farfromrefug** [comment]
I actually have an issue related to that. Because Aec3 and dts are only available in different modes, you have to switch every time you open  a file with one or the other.
I must say that it works.! But couldn't there be a setting for Spmc to automatically switch based on the played file?

**Comment by nvlashi** [comment]
Agree, you can make them work by switching back and forth between the modes but that is a pain.  In the Jarvis version (16.7.3) you can enable all of them so seems specific to this version.

**Comment by lartomar2002** [comment]
I believe this has since been corrected.

**Comment by nvlashi** [comment]
Fixed as of what build?

**Comment by lartomar2002** [comment]
I am using 17.6a and not experiencing any problems.
